### ISSUE TITLE
Vagrant failed to initialize at a very early stage after upgrade to 2.2.5

### ISSUE THREAD

**Issue by Wenzel** [hashicorp/vagrant#11058]
## Vagrant version
Vagrant 2.2.5

### Host operating system
Fedora 30

### Guest operating system
not relevant

### Vagrantfile
not relevant

### Debug output
[gist](https://gist.github.com/Wenzel/b6c4102678bca442946b0fa16082e1b7)

### Expected behavior
`vagrant version` should have worked.

### Actual behavior
~~~
Vagrant failed to initialize at a very early stage:

VirtualBox is complaining that the kernel module is not loaded. Please
run `VBoxManage --version` or open the VirtualBox GUI to see the error
message which should contain instructions on how to fix this error.
~~~

### Steps to reproduce
1. upgrade from `2.2.3` to `2.2.5` using Fedora packages
2. run `vagrant version`
3. it fails

Here is my plugin list:
~~~
$ vagrant plugin list
vagrant-libvirt (0.0.45, system)
vagrant-reload (0.0.1, global)
  - Version Constraint: > 0
~~~

I did run `vagrant plugin update` after the package upgrade, it just worked.
But then `vagrant version/up/etc` fails.

I'm using vagrant-libvirt with QEMU/KVM, I don't really care about VirtualBox.
It was working fine before, why does it complain now ?
This brakes my workflow :(

Thanks Vagrant Team !

**Comment by Wenzel** [comment]
I had to downgrade vagrant to `2.2.3` to get it working:
~~~
 ⚙ wenzel@strix  ~  vagrant version
Vagrant failed to initialize at a very early stage:

VirtualBox is complaining that the kernel module is not loaded. Please
run `VBoxManage --version` or open the VirtualBox GUI to see the error
message which should contain instructions on how to fix this error.
 ✘ ⚙ wenzel@strix  ~  sudo dnf downgrade vagrant
[sudo] Mot de passe de wenzel : 
Dernière vérification de l’expiration des métadonnées effectuée il y a 0:30:53 le dim. 08 sept. 2019 12:15:46 CEST.
Dépendances résolues.
=============================================================================================================================================================================================
 Paquet                                      Architecture                               Version                                             Dépôt                                      Taille
=============================================================================================================================================================================================
Rétrogradation:
 vagrant                                     noarch                                     2.2.3-1.fc30                                        fedora                                     584 k

Résumé de la transaction
=============================================================================================================================================================================================
Retrograder  1 Paquet

Taille totale des téléchargements : 584 k
Voulez-vous continuer ? [o/N] : o
Téléchargement des paquets :
vagrant-2.2.3-1.fc30.noarch.rpm                                                                                                                              3.1 MB/s | 584 kB     00:00    
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Total                                                                                                                                                        807 kB/s | 584 kB     00:00     
Test de la transaction
La vérification de la transaction a réussi.
Lancement de la transaction de test
Transaction de test réussie.
Exécution de la transaction
  Préparation           :                                                                                                                                                                1/1 
  Exécution du scriptlet: vagrant-2.2.3-1.fc30.noarch                                                                                                                                    1/1 
  Exécution du scriptlet: vagrant-2.2.3-1.fc30.noarch                                                                                                                                    1/2 
  Rétrogradation        : vagrant-2.2.3-1.fc30.noarch                                                                                                                                    1/2 
  Exécution du scriptlet: vagrant-2.2.3-1.fc30.noarch                                                                                                                                    1/2 
  Nettoyage de          : vagrant-2.2.5-1.fc30.noarch                                                                                                                                    2/2 
  Exécution du scriptlet: vagrant-2.2.5-1.fc30.noarch                                                                                                                                    2/2 
  Exécution du scriptlet: vagrant-2.2.3-1.fc30.noarch                                                                                                                                    2/2 
  Vérification de       : vagrant-2.2.3-1.fc30.noarch                                                                                                                                    1/2 
  Vérification de       : vagrant-2.2.5-1.fc30.noarch                                                                                                                                    2/2 

Rétrogradé:
  vagrant-2.2.3-1.fc30.noarch                                                                                                                                                                

Terminé !
 ⚙ wenzel@strix  ~  vagrant version
==> vagrant: A new version of Vagrant is available: 2.2.5 (installed version: 2.2.3)!
==> vagrant: To upgrade visit: https://www.vagrantup.com/downloads.html

Installed Version: 2.2.3
Latest Version: 2.2.5
 
To upgrade to the latest version, visit the downloads page and
download and install the latest version of Vagrant from the URL
below:

  https://www.vagrantup.com/downloads.html

If you're curious what changed in the latest release, view the
CHANGELOG below:

  https://github.com/hashicorp/vagrant/blob/v2.2.5/CHANGELOG.md
 ⚙ wenzel@strix  ~  
~~~

**Comment by Wenzel** [comment]
@briancain if you need more information, I can provide, the bug is 100% reproductible.

Thanks.

**Comment by dukebarman** [comment]
After upgrade to 2.2.6 I have a same issue. OS: macOS 10.14.6

**Comment by dukebarman** [comment]
I reuninstall virtualbox and problem was disappeared.

**Comment by chrisroberts** [comment]
It looks like this error was being generated because VirtualBox was installed on the system, but the modules were not installed which caused the provider check on VirtualBox to fail (the `VBoxManage --version`). Since this is just for a provider check, failure of the command should just remove the provider from the list of available providers on the system. Wrapping some error checking around the command should resolve this.

As a workaround, fixing the installed version of VirtualBox, or removing it, should resolve the issue. I believe explicitly defining the provider will also resolve it until a fix is in place.

**Comment by chrisroberts** [comment]
Fixed via #10938
